### PR TITLE
Added my readme file

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**

### CODE REVIEW

File: Readmes/Mahendra/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- 🔭 I’m currently working on **Personal Projects.**
+
+- 🌱 I’m currently learning **Something New 😉.**
+
+- 👯 I’m looking to collaborate on **Cool Projects.**
+
+- 💬 Ask me about **JavaScript.🤩**
+
+- 📫 How to reach me **mahendrabishnoi4488@gmail.com**
```
Comment (Panquesito7) [inline Readmes/Mahendra/README.md:15]:
```suggestion
- 📫 How to reach me: **mahendrabishnoi4488@gmail.com**
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant

File: Readmes/Mahendra/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# 📊GitHub Stats :
```
Comment (Panquesito7) [inline Readmes/Mahendra/README.md:25]:
```suggestion
# 📊GitHub Stats
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inelegant

File: Readmes/Mahendra/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+# 📊GitHub Stats
+
+![](https://github-readme-stats.vercel.app/api?username=MahendraBishnoi29&theme=dark&hide_border=false&include_all_commits=false&count_private=true)<br/>
+![](https://github-readme-streak-stats.herokuapp.com/?user=MahendraBishnoi29&theme=dark&hide_border=false)<br/>
+
+---
+
+![](https://komarev.com/ghpvc/?username=MahendraBishnoi29&label=Visitors+Count&color=brightgreen)
```
Comment (Panquesito7) [inline Readmes/Mahendra/README.md:32]: Remove the profile count badge, as it's inaccurate.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Comment by MahendraBishnoi29** [comment]
Hey @Panquesito7 it's not merged yet is anything wrong with my PR ?

**Review by Greeshma2903 — APPROVED** [review]

**Pull request merged**